scene/gui/tab_container.cpp

```diff
@@ -143,7 +143,14 @@ void TabContainer::_notification(int p_what) {
 			}
 		} break;
 
-		case NOTIFICATION_READY:
+		case NOTIFICATION_READY: {
+			if (setting_up) {
```

Comment:
`NOTIFICATION_READY` is only received once per node, so this won't work correctly when adding the node multiple times.

Reply:
I can use `NOTIFICATION_POST_ENTER_TREE` instead.

Reply:
I do need to check the `setting_up` here, otherwise `set_current_tab` will be called even if it wasn't called out of the tree. We don't want this because it can send a signal, and in when `current_tab` is its default of `0` it would call it and cause an index out of range error if there are no children.